distributed/deploy/local.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import print_function, division, absolute_import
+
+from threading import Thread
+
+from tornado.ioloop import IOLoop
+from tornado.iostream import StreamClosedError
+from tornado import gen
+
+from ..utils import sync, ignoring, All
+from ..executor import Executor
+from ..nanny import Nanny
+from ..scheduler import Scheduler
+from ..worker import Worker, _ncores
+
+
+class Local(object):
+    def __init__(self, n_workers=None, cores_per_worker=None, processes=True, loop=None, start=True, scheduler_port=8786, **kwargs):
```

Comment:
So this is a bit confusing.  There are three relevant keyword arguments in this decision:

1.  n_workers
2.  cores_per_worker: similar to `--nthreads` in `dask-worker`
3.  processes: similar to `--nanny` in `dask-worker`.  Switches between creating a Worker (threadpool in this process) and a Nanny (threadpool in other process)

So if n_workers was 1 and cores_per_worker was 4 and processes was true this would start a four-threaded worker in a separate process (using a nanny).  Can you recommend better names for these keyword arguments?